source/sweetsourcod/lempel_ziv.hpp

```diff
@@ -25,8 +25,8 @@ std::string int_vector_to_string(const std::vector<T>& v){
     ss.reserve(v.size());
     for (const auto& x: v){
         if (x < 0) {throw std::runtime_error("int_vector_to_string only accepts sequences of positive values");}
-        if (x > 255) {throw std::runtime_error("x>255, exceeded ascii table");}
-        ss.push_back(static_cast<int>(x));
+        if (x > 254) {throw std::runtime_error("x>255, exceeded ascii table");}
```

Comment:
let's write 2 versions of this function, one for 1 to 255 to be use with the cross parser and one for 0-255 to be used with the regular LZ